src/sas/qtgui/Perspectives/Fitting/FittingPerspective.py

```diff
@@ -272,6 +285,38 @@ def addFit(self, data, is_batch=False, tab_index=None):
         # Notify listeners
         self.tabsModifiedSignal.emit()
 
+    def nextAvailableTabIndex(self):
+        """
+        Returns the index of the next available tab
+        """
+        max1 = max([tab.tab_id for tab in self.tabs], default=0)
+        if self.lastTabClosed is None:
+            return max1 + 1
+        max2 = self.lastTabClosed.tab_id
```

Comment:
No need for a second return:

`max2 = 0 if self.lastTabClosed is None else self.lastTabClosed.tab_id`